.gitignore

```diff
@@ -1,8 +1,11 @@
 node_modules
 .eslintcache
+.vscode
 user-build-config.yml
 *.log
 .idea
 user-config.*
 dev-config.*
 /artifacts/
+/node_modules
```

Comment:
Agreed, let's remove `/node_modules`. This is covered by the first line `node_modules`.

Reply:
alright i will fix that